### PR TITLE
BaSP-TG-26: test update and delete superadmins

### PR DESCRIPTION
**Pull request by JuliMonte5**
Tests for update and delete methods of superadmins feature.

### CODE REVIEW

**Comment by sofihoj** [comment]
The name of the PR should follow the format BaSP-TG-XX: Short description

**Comment by agustin-schiaffino** [comment]
maybe you could add some tests to check the messages of the responses in case of success and error, you can look into some of the PRs that were merge to guide you

File: src/tests/super-admins.test.js [hunk 1]
```diff
@@ -101,6 +117,11 @@ describe('PUT /super-admins', () => {
     const response = await request(app).put(`/super-admins/${reqId}`).send(mockedBadSuperAdmin);
     expect(response.body.data).toBe(undefined);
   });
+
+  test('check for error message', async () => {
+    const response = await request(app).put(`/super-admins/${reqId}`).send(mockedBadSuperAdmin);
+    expect(response.body.message).toBeDefined();
```
Comment (agustin-schiaffino) [inline src/tests/super-admins.test.js:123]: here I think is better to check for the content of the mesage instead, please try to do it

File: src/tests/super-admins.test.js [hunk 2]
```diff
@@ -149,4 +180,9 @@ describe('DELETE /super-admins', () => {
     const response = await request(app).delete(`/super-admins/${badReqId}`).send();
     expect(response.body.data).toBe(undefined);
   });
+
+  test('check for error message', async () => {
+    const response = await request(app).delete(`/super-admins/${badReqId}`).send();
+    expect(response.body.message).toBeDefined();
```
Comment (agustin-schiaffino) [inline src/tests/super-admins.test.js:186]: same as line 123

**Review by Alfondalix — APPROVED** [review]
Well done

**Pull request merged**